cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/scheduler/SchedulerQueueResolver.java

```diff
@@ -61,6 +62,9 @@ public String getDefaultQueue() {
    */
   @Nullable
   public String getQueue(Id.Namespace namespaceId) throws IOException, NamespaceNotFoundException {
+    if (namespaceId.equals(NamespaceId.SYSTEM.toId())) {
```

Comment:
For system namespace, I think it should always use the cdap scheduler queue.

https://github.com/caskdata/cdap/blob/develop/cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java#L909